fuzzers/kirenenko/builder.Dockerfile

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# # http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG parent_image=gcr.io/fuzzbench/base-builder
+FROM $parent_image
+#FROM ubuntu:16.04
+
+# Install Clang/LLVM 6.0.
+RUN apt-get update -y && \
+    apt-get -y install libc6 libstdc++6  \
+		linux-libc-dev linux-libc-dev gcc-multilib \
+    llvm-6.0 clang-6.0 llvm-6.0-dev llvm-6.0-tools \
+		llvm-dev  zlib1g-dev git cmake  python-pip \
+		g++ g++-multilib wget
+
+RUN apt-get install -y libstdc++-5-dev libtool-bin automake flex bison \
+                       libglib2.0-dev libpixman-1-dev python3-setuptools unzip \
+                       apt-utils apt-transport-https ca-certificates
+
+
+#download install AFL++
```

Comment:
nit: we begin comments with a capital letter and end them with a punctuation mark.
Please change to "# Download and install AFL++."